### PR TITLE
Add emojis to the commit messages in contributions

### PR DESCRIPTION
**Pull request by dialelo**
Adds some emoji :green_heart: to the commit messages in the list of contributions. A bit dull right now with only one commit with the ":baby:" message so make sure to include :cool: emojis in your commit messages :sunglasses: 

baby

### CODE REVIEW

**Comment by rafaparadela** [comment]
🍻 !! LGTM

Note: IMO 🍻  is the emoji that best represents the awesomeness.

File: server/app/views/templates/library/index.scala.html [hunk 1]
```diff
@@ -125,6 +125,7 @@ <h2 class="pull-left">@section.name.humanizeUnderscore</h2>
       <!-- JS -->
     <script src="@routes.Assets.at("lib/highlightjs/highlight.min.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("lib/highlightjs/languages/scala.min.js")" type="text/javascript"></script>
+    <script src="@routes.Assets.at("javascripts/emojify-cdn.js")"></script>
```
Comment (rafaparadela) [inline server/app/views/templates/library/index.scala.html]: What do you think about including this plugin with webjars `"org.webjars.npm" % "emojify.js" % "1.1.0"` ? @dialelo looks good to you?

Comment (dialelo) [inline server/app/views/templates/library/index.scala.html]: This is not the emojify library but emojify-cdn.

Comment (rafaparadela) [inline server/app/views/templates/library/index.scala.html]: Ok

**Comment by dialelo** [comment]
:shipit:

**Pull request merged**